contracts/core/helpers/MainnetCoreAddresses.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.7.6;
+
+contract MainnetCoreAddresses {
+    address internal constant DEFI_SAVER_LOGGER_ADDR = 0x5c55B921f590a89C1Ebe84dF170E655a82b62126;
+    address internal constant REGISTRY_ADDR = 0xD6049E1F5F3EfF1F921f5532aF1A1632bA23929C;
+    address internal constant PROXY_AUTH_ADDR = 0x3Aa5ebB10DC797CAC828524e59A333d0A371443c;
```

Comment:
Not a mainnet address, probably injected while testing, put a placeholder addr here